packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap

```diff
@@ -416,285 +416,285 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
+--steplist-step__border-color: var(--color-border);
+--steplist-step__color--completed: var(--color-base);
+--steplist-step__color--current: var(--color-white);
+--steplist-step__color: var(--color-muted);
+--steplist__background-color--current: var(--color-primary);
+--steplist__color--current: var(--color-primary);
+--steplist__color: var(--color-muted);
+--table-header__background-color: var(--color-primary-lightest);
+--table__background-color--striped: var(--color-primary-lightest);
+--table__border-color: var(--color-gray-lighter);
+--table__padding: var(--spacer-2);
+--tabs-panel__background-color: var(--color-background);
+--tabs__background-color--disabled: var(--color-gray-lighter);
+--tabs__background-color--hover: var(--color-primary-lightest);
+--tabs__background-color--selected: var(--color-background);
+--tabs__background-color: var(--color-background);
+--tabs__border-color--disabled: var(--color-gray-lighter);
+--tabs__border-color--selected: var(--color-primary);
+--tabs__border-color: var(--color-border);
+--tabs__color--active: var(--color-primary-darker);
+--tabs__color--disabled: var(--color-gray-darker);
+--tabs__color--hover: var(--color-primary);
+--tabs__color--selected: var(--color-primary);
+--tabs__color: var(--color-base);
+--text-input-currency-icon__line-height: var(--text-input__line-height);
 --text-input-currency-icon__spacing: 20px;
---text-input__background-color--disabled: #d9d9d9;
---text-input__border-color--disabled: #a6a6a6;
---text-input__border-color--error--inverse: #f18e9e;
---text-input__border-color--error: #e31c3d;
---text-input__border-color--inverse: #000000;
---text-input__border-color: #262626;
---text-input__border-radius: 3px;
+--text-input__background-color--disabled: var(--color-gray-lighter);
+--text-input__border-color--disabled: var(--color-gray-light);
+--text-input__border-color--error--inverse: var(--color-error-light);
+--text-input__border-color--error: var(--color-error);
+--text-input__border-color--inverse: var(--color-black);
+--text-input__border-color: var(--color-base);
+--text-input__border-radius: var(--radius-default);
 --text-input__border-width--disabled: 2px;
 --text-input__border-width--error: 3px;
 --text-input__border-width: 2px;
---text-input__color: #262626;
---text-input__divider-color: #a6a6a6;
+--text-input__color: var(--color-base);
+--text-input__divider-color: var(--color-gray-light);
 --text-input__line-height: 1.2999999523162842;
---text-input__padding: 8px;
---tooltip-icon__color--inverse: #ffffff;
---tooltip-icon__color: #0d2499;
---tooltip-trigger__color: #0d2499;
---tooltip__background-color: #ffffff;
+--text-input__padding: var(--spacer-1);
+--tooltip-icon__color--inverse: var(--color-white);
+--tooltip-icon__color: var(--color-primary);
+--tooltip-trigger__color: var(--color-primary);
+--tooltip__background-color: var(--color-background);
 --tooltip__border-color--active: #0d249940;
 --tooltip__border-color--inverse-active: #ffffff40;
```

Comment:
did these vars not have another variable?

Reply:
Not sure what's going on here. It should be referencing `--color-gray-darker`

Reply:
Oh, that's the `--tooltip__border-color`

Reply:
Oh yeah, these are hard-coded color values because Figma doesn't know how to take a base color from a variable and add transparency. I think we're just going to have to live with this for now. TODO.